tests/Integration/OrderBy/OrderByDirectiveDBTest.php

```diff
@@ -15,6 +15,8 @@ class OrderByDirectiveDBTest extends DBTestCase
             orderByRestricted: _ @orderBy(columns: ["name"])
             orderByRestrictedEnum: _ @orderBy(columnsEnum: "UserColumn")
         ): [User!]! @all
+
+        latestUsers: [User!]! @all @orderBy(column: "created_at" direction: "DESC")
```

Comment:
I am trying to move away from having a single large schema per test class, it is rather bad for maintenance. Please define it down in the individual test method instead.